hue/hue/base/params.env

```diff
@@ -0,0 +1,4 @@
+storage_class=
+s3_endpoint=
+s3_region=
+s3_is_secure=false
```

Comment:
It is missing the parameters for the S3 keys

Reply:
let's discuss this at https://github.com/opendatahub-io/odh-manifests/pull/228#discussion_r521379415 it's the same thing.

Reply:
resolved via

```
s3_credentials_secret=thriftserver-sample-s3
```